apps/web/lib/features/integrations/calendar/table-time-sheet.tsx

```diff
@@ -183,28 +80,7 @@ export function DataTableTimeSheet({ data, user }: { data?: GroupedTimesheet[],
 	};
 
 	const t = useTranslations();
-	const [sorting, setSorting] = React.useState<SortingState>([]);
```

Comment:
@Innocent-Akim why that whole block was added and now removed? What was function of it? Did you replaced it with something else or it was not needed at all or what?

Reply:
@evereq  It's almost the same, except that when I used the hook to dispatch the data to the table, I analyzed how to reuse the same code. I realized it was impossible because it was tailored to the Shadcn table. Then, I understood that it was very important to remove it to make the code clearer.